ironfish/src/account/accounts.ts

```diff
@@ -630,10 +642,21 @@ export class Accounts {
         }
 
         if (!map.spent) {
+          let confirmedMinimumBlocks = false
```

Comment:
nit: can we just call this something simple like confirmed?

Reply:
Works for me, updated.